### DISCUSSION TITLE
[Unstable] Mechanic talent feedback v0.20.0.x

### DISCUSSION THREAD

**Discussion by TwitchyTwigger** [FakeFishGames/Barotrauma#10274] [Unstable feedback]
Mechanic has needed some extra flavor for awhile now, and this talent rework adds a lot to the role, however there are some issues I'd like to bring up.

But first, general talent concerns:

Right now, the talent system is way too limited and implies the need for 3 crewmember of each class to have access to all the game's content, that comes to 17 crewmembers if we assume Captain still only gets 2 talent trees, the highest recommended crew submarine currently is the Berilia. with a 12-16 crew recommendation.

Realistically, I think, the core talent progression should be anywhere from 3 to unlimited 'general' talents and 2 talent trees, that way there's still reason to have multiple of each role while not being as harshly limiting.

And now on to Mechanic specific talents:

1. Ironman: honestly a confusing talent, I wouldn't trust the maintenance money to supply armor considering there's nicer, Kevlar options available.
 
2. Tool maintenance: Should only allow the Mechanic themselves to craft at a higher quality, as their main job other than maintaining the submarine is to supply the crew with crafting, seems weird to encourage everyone to craft on the Mechanic's behalf.
 
3. Tinkerer: Man is it massively disheartening to lose Mechanic's active ability, being asked by the Captain or Gunners to boost their components was really fun and added to the team dynamic, the "I'm giving her all she's got" aspect is just gone now.
 
4. Mudraptor Wrestler: Very situational, only affects one enemy in the game, In my opinion it should be replaced by a talent that gives you a 50% melee damage bonus to anything inside the submarine itself, a decent self defense talent for when something's between you and that busted pump.

There's also a couple new Mechanic items that might need a little tweaking:

1. The Scrap Cannon: Rather underwhelming when you consider that it uses 2 ammo types (scrap/ores and batteries) and has generally worse damage than a basic shotgun, it would be neat if different ores applied different damage types, and if the spread was toned down a little.

It's also worth noting that batteries are a weird choice, using oxygen tanks as compressed air for launching the ammo seems like it'd make more sense, and also a charge mechanic could be added that increases projectile velocity/damage and reduces spread at the cost of using more oxygen from its tank.

2. The Exosuit: This thing is badass but it had one really goofy aspect: The ability to swim.

This thing's a lumbering, personal tank, and it swims just like a dude in a suit. It would be a really unique feature if the Exosuit could walk along the sea floor and have a hydropack/jetpack powered by batteries/welding fuel for vertical maneuvering. Also could use some brighter lights.

And finally some questionable swapping of talents to other roles:

1. Pyromaniac: Why would an Engineer ever have to interact with Ballast Flora? Pumps are a mechanical component it makes no sense for an Electrical Engineer to have a talent catered to dealing with them.

2. Scavenger: Why would a Security Officer be looting anything? Their job is to bust heads not scavenge for trinkets, that's what EVA/mining Mechanics used to do, come in after Security secured the area and loot everything that's not nailed down.

And that concludes my concerns regarding the talent rework, I hope some or more of these are looked into as there's a lot of potential here, just muddied by a few questionable/confusing choices.

**Comment by Instant2116** [comment]
I agree with most of the feed back and would like to add suggestion about Exosuit :  if Exosuit will use it's own state to take damage (Exosuit  helth) with same damage resists, once state reach 0% Exosuit  won't provide Pressure Resistance and can be repaired (could be additional condiotions like "can not repair Exosuit  while in use" and "Required Mechanic Skill 95").
P.S. This message will be duplicated in different platforms to help Exosuit to entice instead of estrange people. As by opinion of the author.
